drivers/spi/spi_context.h

```diff
@@ -245,6 +245,20 @@ bool spi_context_rx_on(struct spi_context *ctx)
 	return !!(ctx->rx_buf || ctx->rx_len);
 }
 
+static ALWAYS_INLINE
+size_t spi_context_longest_single_xfer_buffers_len(struct spi_context *ctx)
+{
+	if (ctx->tx_len == 0) {
```

Comment:
follow the style applied in this file, so if (!ctx->tx_len) { ...

Apply that also for rx_len test